pages/community/styleguide/aboutguide.md

```diff
@@ -0,0 +1,35 @@
+# AsyncAPI Style Guide
+
+## Introduction
+Welcome to the AsyncAPI Style Guide. This page serves as a guideline to contributors on how to write, structure, and format content/documentation.
```

Comment:
```suggestion
Welcome to the AsyncAPI Style Guide. This page guides contributors on how to write, structure, and format content in documentation.
```

We shouldn't use `/` in technical docs, that's informal so I would use it for blog posts.